libclamav_rust/src/fuzzy_hash.rs

```diff
@@ -36,7 +36,7 @@ use rustdct::DctPlanner;
 use thiserror::Error;
 use transpose::transpose;
 
-use crate::{frs_error::FRSError, frs_result, sys, validate_str_param};
+use crate::{frs_call, frs_error, frs_error::FRSError, sys, validate_str_param};
```

Comment:
I'm a little surprised that Rust isn't confused between `crate::frs_error` (the macro) and `crate::frs_error` the module.  I worry we should rename the macro or the module to make it less confusing.